VTEX - Catalog API.json

```diff
@@ -5102,7 +5102,7 @@
                                     "PackagedWeightKg": {
                                         "type": "integer",
                                         "title": "PackagedWeightKg",
-                                        "description": "Weight used for shipping calculation, in the measurement [configured in the store](https://help.vtex.com/en/tutorial/filling-in-system-settings--tutorials_269), which by default is in grams.",
+                                        "description": "Weight used for shipping calculation, in the measurement [configured in the store](https://help.vtex.com/en/tutorial/filling-in-system-settings--tutorials_269), which by default is in grams. Don't fill in this field with 0 or null, because this might result in shipping issues.",
```

Comment:
```suggestion
                                        "description": "Weight used for shipping calculation, in the measurement [configured in the store](https://help.vtex.com/en/tutorial/filling-in-system-settings--tutorials_269), which by default is in grams. Do not fill in this field with `0` or `null`, because this might result in shipping issues.",
```